scripts/external_libs/pyzmq-ctypes/zmq/bindings.py

```diff
@@ -11,7 +11,15 @@
 from zmq.constants import *
 from ctypes import *
 
-cpu_vendor = 'arm' if os.uname()[4] == 'aarch64' else 'intel'
+march = os.uname()[4]
+
+if march == 'aarch64':
+    cpu_vendor = 'arm'
+elif march == 'x86_64' or march == 'i386' or march == 'i686':
+    cpu_vendor = 'intel'
+else:
+    assert(cpu_vendor, 'Unknown CPU architecture')
```

Comment:
This assert will never raise Exception

Reply:
Changed the assert to "raise Exception()".  Verified the code by commenting out the ppc64le support and running the functional regression test ("./run_regression --func") which generated the expected error.